charts/plex-media-server/README.md

```diff
@@ -0,0 +1,126 @@
+# plex-media-server Chart
+===========
+
+A Helm chart for deploying the a Plex Personal Media Server(PMS) server.
+
+This is not an officially supported Plex product. This project is actively being maintained.
```

Comment:
This might be confusing coming from a repo that Plex owns. Can we reword this to something like

>While Plex is responsible for maintaining this Helm chart, we cannot provide support for troubleshooting related to its usage. For community assistance, please visit our support forums.

Reply:
yup I can reword that, I took the wording from one of the google projects I have used a couple times. Your wording is definitely clearer